pandas/core/series.py

```diff
@@ -862,7 +862,9 @@ def __getitem__(self, key):
 
                 return result
             except InvalidIndexError:
-                pass
+                if not isinstance(self.index, MultiIndex):
```

Comment:
What is this for?

Reply:
it wasn't obvious to me that non-MI shouldn't be allowed through here, this should clarify it for future readers (or me once I forget)